qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -398,7 +404,7 @@ def evaluate_energy(parameters: np.ndarray) -> np.ndarray | float:
                     self._eval_count += 1
                     self.callback(self._eval_count, params, value, meta, step)
             else:
-                self._eval_count += len(values)
+                self._eval_count += len(values) / batch_size
```

Comment:
I think the number of evaluations usually counts the number of function calls, so batching wouldn't change this -- but maybe double check to be the same as VQE 🙂